runtime/legion/legion_ops.cc

```diff
@@ -8027,6 +8195,8 @@ namespace Legion {
       // Map this is a restricted region. We already know the 
       // physical region that we want to map.
       InstanceSet mapped_instances = restrict_info.get_instances();
+      // TODO: Implement physical tracing for acquire operation
```

Comment:
Another acquire for the github issue list

Reply:
I think some of these comments do not make sense any more. I'll change them to a list of operations the tracing does not (and will not) support.